Starcounter.Uniform.Tests/Queryables/QueryableFilterTests.cs

```diff
@@ -54,5 +55,129 @@ public void ApplyPartialFilterShouldReturnProperRowDatas()
             returnedData.Should().HaveCount(2);
             returnedData.Should().NotContain(x => x.Name == "Tom");
         }
+
+        [Test]
+        public void ApplyingMultipleFiltersShouldReturnProperRowsDatas()
+        {
+            var data = new List<RowDataModel>
```

Comment:
you reuse this data in all the tests. Why not extract it to setup?

Reply:
actually, you could generalize into code that accepts filter configuration and expected results and runs all the test